lib/routes/content.js

```diff
@@ -152,7 +246,17 @@ const routes = application => {
       }
 
       return true;
-    });
+    };
+
+    /*
+     * @name Add - Post to Content Type
+    */
+    app.post(`/${config.content.base}/:type/${config.content.actions.save}`, multipartMiddleware, save);
```

Comment:
I literally don't understand why you are making multiple endpoints that do the exact same thing. They should all be calling `content/save`, that's why I made it like that